vitest.config.ts

```diff
@@ -2,6 +2,7 @@ import { defineConfig } from 'vitest/config'
 
 export default defineConfig({
   test: {
+    include: ['**/__tests__/**/*.spec.[tj]s'],
```

Comment:
The latest version of Vitest will treat any matched files without `.spec` and `.test` as test files. This brings back the restriction we want.